### PR TITLE
importlib_metadata v0.5

### PR DESCRIPTION
**Pull request by nehaljwani**
New requirement of path.py

### CODE REVIEW

File: recipes/importlib_metadata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - importlib_metadata
+
+about:
+  home: The package home page
```
Comment (synapticarbors) [inline recipes/importlib_metadata/meta.yaml]: Please add a proper entry for `home`

File: recipes/importlib_metadata/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  license: Apache Software
```
Comment (synapticarbors) [inline recipes/importlib_metadata/meta.yaml]: This looks like it should be `license: APACHE-2.0`

File: recipes/importlib_metadata/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f6fb31513ab8cf7a1fceb5d67041989f05c0c3c3a54998328e257943627121e3
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```
Comment (synapticarbors) [inline recipes/importlib_metadata/meta.yaml]: Please update to latest standard for `script`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33

File: recipes/importlib_metadata/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+test:
+  imports:
+    - importlib_metadata
+
+about:
+  home: The package home page
+  license: Apache Software
+  license_family: APACHE
+  license_file: LICENSE
+  summary: Summary of the package
```
Comment (synapticarbors) [inline recipes/importlib_metadata/meta.yaml]: Please add proper entry for `summary`

File: recipes/importlib_metadata/meta.yaml (hunk 3 above)
Comment (nehaljwani) [reply recipes/importlib_metadata/meta.yaml]: I deliberately dropped --no-cache-dir . More discussion at https://github.com/conda/conda-build/issues/3094

Comment (synapticarbors) [reply recipes/importlib_metadata/meta.yaml]: ok, thanks for pointing me to that.

**Pull request merged**